core/root/cluster.go

```diff
@@ -348,3 +366,43 @@ func (c clusterImpl) ValidateStack() (string, error) {
 
 	return strings.Join(reports, "\n"), nil
 }
+
+func printJournaldLogs(c clusterImpl, quit chan bool) error {
```

Comment:
Nit but I prefer `streamJournaldLogs` as it continuously print logs again and again(=stream)